CITATION.cff

```diff
@@ -78,13 +78,13 @@ authors:
     given-names: Stefan
     orcid: 0000-0002-5821-2060
 
-cff-version: "1.0.3"
-date-released: 2021-04-21
+cff-version: "1.0.4"
```

Comment:
Dont think that cff-version exists. See https://github.com/citation-file-format/citation-file-format/tree/main/examples
```suggestion
cff-version: "1.0.3"
```